golang-github-osbuild-composer.spec

```diff
@@ -39,9 +45,19 @@ Provides: lorax-composer
 %{common_description}
 
 %prep
-%goprep
+%forgeautosetup -p1
 
 %build
+GO_BUILD_PATH=$PWD/_build
```

Comment:
wow, this is considerably more complicated, can we have some comments explaining the approach here?

Reply:
I agree it should be documented, but this is only moving code from the `rhel-8.2` branch over here.